tests/commands/device/devices.spec.ts

```diff
@@ -28,7 +28,7 @@ USAGE
   $ balena devices
 
 OPTIONS
-  -a, --application <application>  application name
+  -a, --application <application>  application name, slug (preferred), numeric ID (deprecated)
```

Comment:
Tests are failing because some lines exceed a certain number of characters:

https://ci.product-os.io/builds/12766#L5fd4a12f:3501
```
      -  "-a, --application <application>  application name, slug (preferred), numeric"
      -  "ID (deprecated)"
      +  "-a, --application <application>  application name, slug (preferred), numeric ID (deprecated)"
```
I wonder if this `HELP_RESPONSE` test is a bit... overkill. Maybe we could have such a test for one CLI command for the help mechanism in general... Just a thought.

Reply:
I was getting this failure seemingly intermittently, it seemed to go away.  Agree, that this is overkill - will remove.